java/build.gradle

```diff
@@ -80,7 +80,10 @@ task alljavadoc(type: Javadoc) {
     destinationDir = file("$buildDir/docs/javadoc")
     options.encoding = 'UTF-8'
     // Where to find source files for the different modules
-    options.addStringOption('-module-source-path', './src/*/src/main/java/:./src/*/build/generated/source/slice/main')
```

Comment:
Using a `:` separator only works on Unix platforms. It's better to use `File.pathSeparator`, which will use the correct separator for the current platform.

Without this fix, Javadoc generation was broken on Windows.